src/server/services/validation-rules.js

```diff
@@ -1,6 +1,5 @@
 
 const srcRules = require('../lib/templateRules.json')
-const srcDropdowns = require('../lib/templateDropdowns.json')
```

Comment:
we might be safe to remove this file from the repo now

Reply:
ooh I hadn't considered that. Yeah, it doesn't get used anywhere else now, so it's definitely safe.

I kind of like it though, because it gives us a git history of changes to dropdown values in a way that is much easier to see than if we had to open up all the old worksheets and inspect them manually.